### PR TITLE
Add data attributes to track history link events

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/AmrwFMJU/22-implement-ga-event-tracking-on-content-history-see-all-updates-and-full-page-history-links

We'd like to understand how users interact with the page history
and updates links on a page, so add the data attributes which
enable tracking and provide the appropriate tracking options

cc @fofr

### CODE REVIEW

**Comment by nickcolley** [comment]
Should these have aria-expanded for assistive technology?

File: app/views/govuk_component/document_footer.raw.html.erb [hunk 1]
```diff
@@ -40,7 +40,11 @@
     <% if history.any? %>
       <div class="change-notes" data-module="toggle">
         <p>
-          <a href="#full-history" data-controls="full-history" data-expanded="false">
+          <a href="#full-history" data-controls="full-history"
+                                  data-expanded="false"
+                                  data-track-category="Content History"
+                                  data-track-action="fullPageHistoryOpened"
+                                  data-track-label="full-history">
```
Comment (fofr) [inline app/views/govuk_component/document_footer.raw.html.erb]: Three different styles here:
* Words with spaces for category
* CamelCase for action
* Hyphenated for label

Could they all be hyphenated?

Comment (fofr) [inline app/views/govuk_component/document_footer.raw.html.erb]: Does this also need `data-module="track-click"` to work?

**Comment by steventux** [comment]
thanks @fofr updated

File: app/views/govuk_component/document_footer.raw.html.erb [hunk 2]
```diff
@@ -40,7 +40,12 @@
     <% if history.any? %>
       <div class="change-notes" data-module="toggle">
         <p>
-          <a href="#full-history" data-controls="full-history" data-expanded="false">
+          <a href="#full-history" data-controls="full-history"
+                                  data-expanded="false"
+                                  data-module="track-click"
+                                  data-track-category="content-history"
+                                  data-track-action="full-page-history-opened"
```
Comment (fofr) [inline app/views/govuk_component/document_footer.raw.html.erb]: This will track on open and close, might be worth saying: "full-page-history-toggled"

**Comment by fofr** [comment]
One minor comment, otherwise 👍

**Pull request merged**